### PR TITLE
make each rubric circle it's own button

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-11320
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-11320](https://instructure.atlassian.net/browse/MBL-11320)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.59% | 93.59% | 0%
**Student** | 92.27% | 92.27% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.63% | 92.63% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 1]
```diff
@@ -33,76 +36,72 @@ class RubricCircleView: UIView {
...
         for i in 0..<count {
             let r = ratings[i]
             let selected = i == (rubric?.selectedIndex ?? 0)
 
+            let button = DynamicButton(frame: CGRect(x: center.x, y: center.y, width: w, height: w))
+            button.tag = i
+            button.addTarget(self, action: #selector(actionButtonClicked(sender:)), for: .touchUpInside)
+            button.layer.cornerRadius = floor( w / 2 )
+            button.layer.masksToBounds = true
+                let title = RubricCircleView.formatter.string(for: r) ?? ""
```
Comment (shichongrui) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Is this just github doing weird indentation?

Comment (bluwave) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: it's off in code, fixed, thanks for pointing this out :hides: 😂

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 2]
```diff
@@ -33,76 +36,72 @@ class RubricCircleView: UIView {
...
-        var center = CGPoint(x: w / 2, y: w / 2)
+        var center = CGPoint(x: 0, y: 0)
 
         for i in 0..<count {
             let r = ratings[i]
             let selected = i == (rubric?.selectedIndex ?? 0)
 
+            let button = DynamicButton(frame: CGRect(x: center.x, y: center.y, width: w, height: w))
+            button.tag = i
+            button.addTarget(self, action: #selector(actionButtonClicked(sender:)), for: .touchUpInside)
```
Comment (vanwagonet) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: I think a11y prefers `.primaryActionTriggered` over `.touchUpInside`.

**Pull request merged**